src/main.rs

```diff
@@ -276,6 +278,69 @@ fn main() {
         }
     };
 
+    if args.edit_page && args.edit_patch {
+        print_error(
+            enable_styles,
+            &anyhow!("Don't edit-patch and edit-page at the same time"),
+        );
+        process::exit(1);
+    }
+
+    if args.edit_patch || args.edit_page {
+        if args.command.is_empty() {
+            if args.edit_patch {
+                print_error(
+                    enable_styles,
+                    &anyhow!(
+                        "
+No command given
+
+Sample usage :
+    tldr --edit-patch <cmd>",
+                    ),
+                );
+            }
+            if args.edit_page {
+                print_error(
+                    enable_styles,
+                    &anyhow!(
+                        "
+No command given
+
+Sample usage :
+    tldr --edit-page <cmd>",
+                    ),
+                );
+            }
+            process::exit(1);
+        }
+        let command = args.command.join("-").to_lowercase();
+        let custom_page_dir = config
+            .directories
+            .custom_pages_dir
+            .as_ref()
+            .map(PathWithSource::path);
```

Comment:
This is now duplicated with the code at the end of `main`. Can you make it so that both places use the same variable?